cxplat/src/cxplat_winkernel/memory_winkernel.c

```diff
@@ -56,3 +57,40 @@ cxplat_free(_Frees_ptr_opt_ void* pointer, cxplat_pool_flags_t pool_flags, uint3
         ExFreePoolWithTag(pointer, tag);
     }
 }
+
+_Must_inspect_result_ cxplat_status_t
+cxplat_initialize_lookaside_list(
+    _Out_ cxplat_lookaside_list_t* lookaside, size_t size, uint32_t tag, cxplat_pool_flags_t pool_flags)
+{
+    if (!NT_SUCCESS(ExInitializeLookasideListEx(
+            lookaside,
+            NULL,
+            NULL,
+            _pool_flags_to_type(pool_flags),
+            EX_LOOKASIDE_LIST_EX_FLAGS_FAIL_NO_RAISE,
+            size,
+            tag,
+            0))) {
+        return CXPLAT_STATUS_NO_MEMORY;
+    } else {
+        return CXPLAT_STATUS_SUCCESS;
+    }
```

Comment:
I recommend the following convention (not sure how many code paths don't follow it yet) where you don't put the `else` if the `if` returns. MsQuic does this (and somehow has automatic validation, I think on Linux builds) for it. It results in slightly cleaner code IMO.
```suggestion
    }
    return CXPLAT_STATUS_SUCCESS;
```
Though, I also do question why we can't return the platform error directly?

Reply:
I agree with the style about not putting the else if the if returns.
The principle being to minimize gratuitous indentation.